src/client/hud.cpp

```diff
@@ -278,6 +278,24 @@ void Hud::drawItems(v2s32 upperleftpos, v2s32 screen_offset, s32 itemcount,
 	}
 }
 
+bool Hud::calculateScreenPos(const v3s16 &camera_offset, HudElement *e, v2s32 *pos)
```

Comment:
can we have some doc about what is true and false ? honestly without reading the whole function (which sounds fine but not very easy to read) it's hard to understand why we return that

Reply:
done